src/main/java/com/mathworks/ci/helper/MatlabBuilderConstants.java

```diff
@@ -36,6 +36,10 @@ public class MatlabBuilderConstants {
     public static final String JUNIT_RESULTS_CHX = "junitChecked";
     public static final String JUNIT_DEFAULT_FILE = "matlab-artifacts/test-reports/junit.xml";
 
+    public static final String HTML_TEST_RESULTS_FOLDER = "htmlTestResults";
+    public static final String HTML_TEST_RESULTS_CHX = "htmlTestResultsChecked";
+    public static final String HTML_TEST_RESULTS_DEFAULT_FOLDER = "matlab-artifacts/test-reports/html";
```

Comment:
perhaps we drop the last html folder and create a default folder of path - matlab-artifacts/test-report to stay consistent with rest of the default html folder paths?

Reply:
Sure, we can remove the extra layer of path from default folder path